docs/content/components/button.mdx

```diff
@@ -136,6 +136,8 @@ with a primary button for the expected action and a secondary button on its left
 
 <Props of={ButtonGroup} />
 
+- **Do** use the same verb tenses for both actions
```

Comment:
WDYT about adding a subheadline like `### Content guidelines` on top of this? :)